test/extended/testdata/statusfail-failedassemble.yaml

```diff
@@ -0,0 +1,19 @@
+kind: BuildConfig
+apiVersion: v1
+metadata:
+  name: failstatus-assemblescript
+spec:
+  source:
+    git:
+      uri: "https://github.com/pi-victor/ruby-hello-world.git"
```

Comment:
don't use a personal repo for this.  clone the repo to disk and provide an override assemble script on disk and then use a binary build.